src/platform/Darwin/PlatformManagerImpl.cpp

```diff
@@ -53,6 +53,27 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack()
     return err;
 }
 
+CHIP_ERROR PlatformManagerImpl::_StartEventLoopTask()
+{
+    dispatch_resume(mWorkQueue);
+
+    return CHIP_NO_ERROR;
+};
+
+CHIP_ERROR PlatformManagerImpl::_StopEventLoopTask()
+{
+    dispatch_sync(mWorkQueue, ^{
```

Comment:
Please document that the `dispatch_sync` is here so that we guarantee serialization of the caller of `StopEventLoopTask` with respect to any tasks that might already be on the queue.  In particular, if we get here while a task is executing, we don't want to return from this call until after that task is done and the queue is actually suspended.